Cargo.toml

```diff
@@ -15,4 +15,7 @@ members = [
 ]
 
 [patch.crates-io]
-halo2_proofs = { git = "https://github.com/filecoin-project/halo2", branch = "serialize-vk-pk" }
+halo2_proofs = { git = "https://github.com/filecoin-project/halo2", branch = "halo2" }
+neptune = { git = "https://github.com/filecoin-project/neptune", branch = "halo2-pick-master-changes" }
```

Comment:
Thanks for pointing this. It will target `halo2` branch of `neptune` eventually, once https://github.com/filecoin-project/neptune/pull/168 will me merged.